src/ucs/sys/topo/base/topo.c

```diff
@@ -31,8 +31,9 @@
 typedef int64_t ucs_bus_id_bit_rep_t;
 
 typedef struct {
-    ucs_sys_bus_id_t bus_id;
-    char             *name;
+    ucs_sys_bus_id_t                bus_id;
+    char                            *name;
+    ucs_topo_device_name_priority_t name_priority;
```

Comment:
this could be just an "unsigned" type with numbers
default (bdf name) is 0
TCP will pass 10
IB will pass 20